tests/command_line/test_find_spots.py

```diff
@@ -377,3 +377,36 @@ def test_find_spots_with_per_image_statistics(dials_data, tmpdir):
     assert (
         b"|   image |   #spots |   #spots_no_ice |   total_intensity |" in result.stdout
     )
+
+
+def test_find_spots_radial_profile(dials_data, tmpdir):
+    result = procrunner.run(
```

Comment:
Could these tests call the `dials.command_line.find_spots.run()` method directly then we can skip the additional `procrunner` dependency/indirection, as well as removing the need to explicitly check for errors?
https://github.com/dials/dials/blob/54cd6499dcd8af113bb138756bffc89297a2463d/command_line/find_spots.py#L239

Reply:
ok, got it